docs/UserGuide.md

```diff
@@ -152,10 +149,12 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/ROLE] [a/ADDRESS] [c/COURSE]
 * Edits the contact at the specified `INDEX`. The index refers to the index number shown in the displayed address book. The index **must be a positive integer** 1, 2, 3, …
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the contact will be removed i.e. adding of tags is not cumulative.
-* You can remove all the contact’s tags by typing `t/` without
-    specifying any tags after it.
-* You can remove a contact's phone number by typing `p/` without specifying any phone number after it.
+* When editing tags, the existing tags of the person will be removed i.e. adding of tags is not cumulative.
+
+<div markdown="span" class="alert alert-primary">
+:bulb: **Tip:** You can remove all the person’s tags by typing `t/` without specifying any tags after it.
+:bulb: **Tip:** You can remove a person's phone number by typing `p/` without specifying any phone number after it.
```

Comment:
Please run Jekyll server locally and check if things render correctly before pushing. Now this lightbulb got on the previous line, which I think is not intended.